app/src/main/java/ch/epfl/sweng/erpa/activities/MyAccountActivity.java

```diff
@@ -0,0 +1,105 @@
+        LinearLayout linearLayout = new LinearLayout(this);
+        ConstraintLayout constraintLayout = new ConstraintLayout(this);
+        TextView textView = new TextView(this);
+        ImageView imageView = new ImageView(this);
+        textView.setText(myAccountButtons.getText());
+        constraintLayout.addView(textView);
+        constraintLayout.addView(imageView);
+        linearLayout.addView(constraintLayout);
+        linearLayout.setOnClickListener(view -> startActivity(new Intent(this, PendingRequestActivity.class)));
+        return linearLayout;
+    }
+
+    @RequiresApi(api = Build.VERSION_CODES.N) @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+//        userProfileProvider.get();
+
+        maListe.forEach(myAccountButtons -> {
+            if(myAccountButtons.isActiveForPlayer() == true ||
+                    myAccountButtons.isActiveForGM() == true){
+                ConstraintLayout myAccountLayout = findViewById(R.id.myAccountLayout);
+                TextView textView = new TextView(this);
+                textView.setText(myAccountButtons.getText());
+                myAccountLayout.addView(textView);
+//                myAccountLayout.addView(createButton(myAccountButtons));
+            }
+        });
+
+        setContentView(R.layout.activity_my_account);
+
+
+
+//        // listener to start pendingRequestActivity
+//        LinearLayout pending_request = findViewById(R.id.launch_pending_request_activity_layout);
+//        pending_request.setOnClickListener(view -> startActivity(new Intent(this, PendingRequestActivity.class)));
+//        // listener to start confirmedGamesActivity
+//        LinearLayout confirmed_games = findViewById(R.id.launch_confirmed_games_activity_layout);
+//        confirmed_games.setOnClickListener(view -> startActivity(new Intent(this, ConfirmedGamesActivity.class)));
+//        // listener to start pastGamesActivity
+//        LinearLayout past_games = findViewById(R.id.launch_past_games_activity_layout);
+//        past_games.setOnClickListener(view -> startActivity(new Intent(this, PastGamesActivity.class)));
+//        // listener to start hostedGamesActivity
+//        LinearLayout hosted_games = findViewById(R.id.launch_hosted_games_activity_layout);
+//        hosted_games.setOnClickListener(view -> startActivity(new Intent(this, HostedGamesActivity.class)));
+//        // listener to start pastHostedGamesActivity
+//        LinearLayout past_hosted_games = findViewById(R.id.launch_past_hosted_games_activity_layout);
+//        past_hosted_games.setOnClickListener(view -> startActivity(new Intent(this, PastHostedGamesActivity.class)));
+//        // listener to start profileActivity
+//        LinearLayout profile= findViewById(R.id.launch_profile_activity_layout);
+//        profile.setOnClickListener(view -> startActivity(new Intent(this, ProfileActivity.class)));
```

Comment:
Please remove this unnecessary comments. If they're useful add a TODO or FIXME with an asignee